### PR TITLE
Use standard {uniform,normal} if params are 0, 1

### PR DESCRIPTION
**Pull request by jacobhinkle**
This just opportunistically uses our unscaled runtime functions instead of the parametrized ones whenever we detect that the input parameters are constant 0 and 1.

I thought nvcc would remove an identity transformation like `(x * 1 + 0)` automatically but the PTX diff shows 8 removed add(0) instructions in `tests/python/test_python_frontend.py::TestNvFuserFrontend::test_nanogpt_mha_dpa`. This is also reflected in the SASS (for sm80).

### CODE REVIEW

**Comment by jacobhinkle** [comment]
!build --diff

**Comment by jacobhinkle** [comment]
Thunder failures are unrelated. The only codediffs are as expected, in `TensorFactoryTest.FactoryBroadcast` and `tests/python/test_python_frontend.py::TestNvFuserFrontend::test_nanogpt_mha_dpa`.

File: csrc/ops/arith.cpp [hunk 1]
```diff
@@ -182,6 +182,28 @@ TensorView* rand(
...
+  if (!pv.hasValue()) {
+    return false;
+  }
+  if (pv.is<int64_t>()) {
+    return pv.as<int64_t>() == i;
+  }
+  if (pv.is<double>()) {
+    return pv.as<double>() == (double)i;
+  }
+  return false;
```
Comment (zasdfgbnm) [inline csrc/ops/arith.cpp:202]:
```suggestion
  return pv.hasValue() && (pv == i).as<bool>();
```

**Review by zasdfgbnm — APPROVED** [review]

File: csrc/ops/arith.cpp (hunk 1 above)
Comment (jacobhinkle) [reply csrc/ops/arith.cpp:202]: That's awesome. Thanks.

**Pull request merged**